src/create_disk.sh

```diff
@@ -325,14 +325,21 @@ install_uefi() {
     # See also https://github.com/ostreedev/ostree/pull/1873#issuecomment-524439883
     /usr/bin/bootupctl backend install --src-root="${deploy_root}" "${rootfs}"
     # We have a "static" grub config file that basically configures grub to look
-    # in the partition labeled "boot".
+    # in the RAID called "md-boot", if it exists, or the partition labeled "boot".
     local target_efi="$rootfs/boot/efi"
     local grubefi=$(find "${target_efi}/EFI/" -maxdepth 1 -type d | grep -v BOOT)
     local vendor_id="${grubefi##*/}"
     local vendordir="${target_efi}/EFI/${vendor_id}"
     mkdir -p "${vendordir}"
 	cat > ${vendordir}/grub.cfg << 'EOF'
-search --label boot --set prefix
+if [ -e (md/md-boot) ]; then
```

Comment:
What do the parenthesis here do?  I tried looking at the GRUB manual but it wasn't obvious.

Reply:
They delimit a volume name.  A file path would be something like `(md/md-boot)/grub2/grub.cfg`.  So this is asking whether `md-boot` exists.